### PR TITLE
Some little fix...

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_vars.yaml_tmpl [hunk 1]
```diff
@@ -563,14 +568,14 @@ vars:
...
-        - name: mapserverproxy
+        - name: mapserverproxy_wms_capabilitie
           display_name: mapserverproxy_wms
           params:
             SERVICE: WMS
             VERSION: 1.1.1
             REQUEST: GetCapabilities
           level: 3
-        - name: mapserverproxy
+        - name: mapserverproxy_wfs_capabilitie
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_vars.yaml_tmpl]: Is an "s" something missing here? "capabilitie" or "capabilities"?

File: doc/developer/debugging.rst [hunk 2]
```diff
@@ -196,7 +196,7 @@ Run:
 
 .. prompt:: bash
 
-   docker-compose --file=docker-compose.yaml --file=docker-compose-dev.yaml up
+   docker-compose --file=docker-compose.yaml --file=docker-compose-dev.yaml up -b
 
 Open the application with on the following path: ``https://<host>/<entry_point>/dev/<interface>.html``.
```
Comment (jwkaltz) [inline doc/developer/debugging.rst]: redundant "with on" -> suggestion: "at the following path:"

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_vars.yaml_tmpl (hunk 1 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_vars.yaml_tmpl]: This was an error, reverted :-)

File: doc/developer/debugging.rst (hunk 2 above)
Comment (sbrunner) [reply doc/developer/debugging.rst]: Thanks :-)

**Pull request merged**